staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -374,11 +374,16 @@ type Devices struct {
 	NetworkInterfaceMultiQueue *bool `json:"networkInterfaceMultiqueue,omitempty"`
 	//Whether to attach a GPU device to the vmi.
 	// +optional
+	// +listType=set
```

Comment:
You can ignore the `list_type_missing` as they are related to SSA feature which we don't support(AFAIK).
For clarity, `// +listType=set` or  `// +listType=atomic` is for slice of integeres,...(scalar elements). You would need to use `+listType=map` for structs.